src/libraries/System.Private.CoreLib/src/System/IO/FileStream.cs

```diff
@@ -130,6 +130,43 @@ public FileStream(string path, FileMode mode, FileAccess access, FileShare share
         }
 
         public FileStream(string path, FileMode mode, FileAccess access, FileShare share, int bufferSize, FileOptions options)
+            : this(path, mode, access, share, bufferSize, options, 0)
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="T:System.IO.FileStream" /> class with the specified path, creation mode, read/write and sharing permission, the access other FileStreams can have to the same file, the buffer size,  additional file options and the allocation size.
```

Comment:
The `T:` prefix isn't necessary in triple slash comments, except to refer to method overloads.
```suggestion
        /// Initializes a new instance of the <see cref="System.IO.FileStream" /> class with the specified path, creation mode, read/write and sharing permission, the access other FileStreams can have to the same file, the buffer size,  additional file options and the allocation size.
```